src/Server/ProcessMessage.idr

```diff
@@ -212,43 +220,74 @@ loadURI conf uri version = do
   let Just (startFolder, startFile) = splitParent fpath
```

Comment:
`startFolder` and `startFile` are not used anymore. did you forget to remove their calculation from here or did you intend to use them below?

Reply:
Oh, just forgot to remove them. If only we had a linter...